app/uk/gov/hmrc/gform/sharedmodel/formtemplate/destinations/Destination.scala

```diff
@@ -248,7 +248,7 @@ case class UploadableDataStoreDestination(
   includeIf: DestinationIncludeIf,
   failOnError: Option[Boolean],
   formId: FormId,
-  version: FormTemplateVersion,
```

Comment:
Why you relaxed the specific type to the string?

Reply:
I'm not sure what this field is for, but it is not a `FormTemplateVersion`